formbuilder/components/builder/EditableField.js

```diff
@@ -55,7 +55,9 @@ class FieldPropertiesEditor extends Component {
             schema={uiSchema.editSchema}
             formData={formData}
             onChange={this.onChange.bind(this)}
-            onSubmit={onUpdate} />
+            onSubmit={onUpdate}>
+            <button type="submit" className="btn btn-info pull-right">Submit</button>
```

Comment:
In order to correct display of date widget, i had to remove a css rule .something . btn { float right}, this rule was used to pull right this submit button.
Instead of using default Form's submit button, i add one myself with 'pull-right' class.